RabbitMq/1.0.0/installer-manifest.json

```
@@ -0,0 +1,8 @@
+{
+    "add-array-option": [
+        {
+            "target": "\\Pyz\\Client\\RabbitMq\\RabbitMqConfig::getSynchronizationQueueConfiguration",
+            "source": "Spryker\\Shared\\AssetExternalStorage\\AssetExternalStorageConfig::ASSET_EXTERNAL_SYNC_STORAGE_QUEUE"
```

```suggestion
            "source": "\\Spryker\\Shared\\AssetExternalStorage\\AssetExternalStorageConfig::ASSET_EXTERNAL_SYNC_STORAGE_QUEUE"
```
I guess it should be fully qualified namespace to constant